[Rust] seems like flatc generates not compilable code

Hi,

I've switched to flatc version 1.12.0 and flatbuffers crate 0.7.0
On previous versions of flatc and runtime everything compliled and woked fine. But now I got tons of errors like this:

error[E0432]: unresolved import `self::flatbuffers::Verifiable` --> cluster/protofiles/src/base_generated.rs:72:9
   |
72 |     use self::flatbuffers::Verifiable;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `Verifiable` in the root

and also such errors:

error[E0405]: cannot find trait `SimpleToVerifyInSlice` in crate `flatbuffers` --> cluster/protofiles/src/struct_generated.rs:596:19
    |
596 | impl flatbuffers::SimpleToVerifyInSlice for CRejectType {}
    |                   ^^^^^^^^^^^^^^^^^^^^^ not found in `flatbuffers`

during compilation of Rust files generated by flatc.

It's look like flatc generates some verification code and assumes that there is special traits for that. But there are no such traits in flatbuffers crate.

May be I do something wrong? Or may be flatc and flatbuffers crate are out of sync?

--- a/rust/flatbuffers/Cargo.toml
+++ b/rust/flatbuffers/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "flatbuffers"
-version = "0.7.0"
+version = "0.8.0"
 edition = "2018"
 authors = ["Robert Winslow <hello@rwinslow.com>", "FlatBuffers Maintainers"]
 license = "Apache-2.0"

